storage/engine/rocksdb/db.cc

```diff
@@ -39,33 +39,84 @@
 extern "C" {
 #include "_cgo_export.h"
 
-struct DBBatch {
+struct DBEngine {
+  rocksdb::DB* const rep;
```

Comment:
Some of the other `DB*` methods also need the `rocksdb::DB*`, such as `DBFlush` and `DBCompactRange`. I've removed this pointer, though, and added an accessor. I'm going to do some more refactoring to see what else can be cleaned up.